lib/fizzbuzz.rb

```diff
@@ -2,7 +2,19 @@ class SuperFizzBuzz
 
   def run(input)
 
-    #Implement your code here
+    if input % 3 == 0 && input % 5 == 0
```

Comment:
this can be combined in an if..elsif or case statement. that way you can avoid the explicit return statements.

you won't see explicit returns a lot in ruby except in the case of early returns.

```ruby
  def fn
    return true if condition
    false
  end
```

Reply:
How could you solve this using ruby's .tap method?
https://ruby-doc.org/core-2.4.2/Object.html#method-i-tap